Dockerfile-alpine.template

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.18
+FROM alpine:{{ env.variant | ltrimstr("alpine") }}
```

Comment:
I think a future goal would be to combine the Alpine and Debian templates into one to ensure that the common bits and flow remain in sync. This can be a future PR or added to this one 🤷.